test/instrumentation/stability-utils.sh

```diff
@@ -29,37 +29,44 @@ stability_check_setup() {
   kube::golang::setup_env
 }
 
-find_files_to_check() {
-  find . -not \( \
-      \( \
-        -wholename './output' \
-        -o -wholename './_output' \
-        -o -wholename './_gopath' \
-        -o -wholename './release' \
-        -o -wholename './target' \
-        -o -wholename '*/third_party/*' \
-        -o -wholename '*/vendor/*' \
-        -o -wholename '*/hack/*' \
-        -o -wholename '*_test.go' \
-        -o -wholename './test/instrumentation/*.go' \
-        \) -prune \
-    \) \
-    \( -wholename '**/*.go' \
-  \)
+function find_files_to_check() {
```

Comment:
the diff is different
```
 diff filesfind.txt filesgit.txt
2829a2830
> staging/src/k8s.io/apiextensions-apiserver/examples/client-go/hack/tools.go
3226a3228,3230
> staging/src/k8s.io/apimachinery/third_party/forked/golang/json/fields.go
> staging/src/k8s.io/apimachinery/third_party/forked/golang/netutil/addr.go
> staging/src/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go
5521a5526,5527
> staging/src/k8s.io/client-go/third_party/forked/golang/template/exec.go
> staging/src/k8s.io/client-go/third_party/forked/golang/template/funcs.go
6012a6019
> staging/src/k8s.io/code-generator/third_party/forked/golang/reflect/type.go
6219a6227
> staging/src/k8s.io/kube-aggregator/hack/tools.go
6822a6831
> staging/src/k8s.io/metrics/hack/tools.go
6915a6925
> staging/src/k8s.io/noderesourcetopology-api/hack/tools.go
7004a7015
> staging/src/k8s.io/sample-apiserver/hack/tools.go
7077a7089
> staging/src/k8s.io/sample-controller/hack/tools.go

```

is this ok?

Reply:
I see, this new `find` doesn't suppress the `third_party` and `hack` folders of the staging repos

Reply:
should we tune the pathspec `Magic signatures` ?

Reply:
ooh, good catch.

Reply:
hmm, I don't know how to spec a glob an negation together.  Have to go AFK for a bit, will investigate